examples/lnclient/pay_ln_address.ts

```diff
@@ -0,0 +1,34 @@
+import "websocket-polyfill"; // required in node.js
+
+import * as readline from "node:readline/promises";
+import { stdin as input, stdout as output } from "node:process";
+
+// ⚠️ Using locally built SDK for testing — run `pnpm build` before running this example.
+import { LN, USD } from "../../dist/index.module.js";
```

Comment:
We want to avoid this as it defeats the whole purpose of moving to typescript. We either need to import the actual typescript files or import from `@getalby/sdk` and add a package.json to the examples folder.